utils/outputwriter/markdowntable.go

```diff
@@ -111,22 +141,51 @@ func (t *MarkdownTableBuilder) Build() string {
 		return ""
 	}
 	var tableBuilder strings.Builder
+	// Calculate Hidden columns
+	for c := range t.columns {
+		// Reset shouldHideColumn flag
+		t.columns[c].shouldHideColumn = t.columns[c].OmitEmpty
```

Comment:
Please explain why you are doing this and how the OmitE,pty value is related to the shouldHideColumn